src/main/webapp/app/exam/manage/students/exam-students.component.scss

```diff
@@ -1,3 +1,5 @@
+@use 'sass:map';
```

Comment:
Same here, we should resolve it in codacy if it is inaccurate

<img width="1581" height="189" alt="image" src="https://github.com/user-attachments/assets/61078665-2cbe-4961-8d1b-3f8ad97f616e" />